src/Tools/dotnet-monitor/DiagnosticsMonitorCommandHandler.cs

```diff
@@ -176,6 +178,8 @@ public static IHostBuilder CreateHostBuilder(IConsole console, string[] urls, st
 
                     services.AddSingleton<IAuthOptions>(authenticationOptions);
 
+                    services.AddSingleton<IEgressOutputOptions>(egressOptions);
```

Comment:
I think this should be moved in here: https://github.com/dotnet/dotnet-monitor/blob/d7c6fcb5e92632b3f4597b1a6bc6437486c00b6b/src/Tools/dotnet-monitor/ServiceCollectionExtensions.cs#L49 because it goes with egress options.

Reply:
It kind of does and kind of doesn't go with the egress system because 'http egress' is not a real egress, but to the user they probably see it the same. So I'm okay either way with it being where it is or where Patrick says to put it. We may refactor it back out later in the future depending on how we want to use the WebApi library.